_inc/client/components/jetpack-notices/index.jsx

```diff
@@ -143,28 +143,27 @@ export const DevModeNotice = React.createClass( {
 	render() {
 		const devMode = window.Initial_State.connectionStatus.devMode;
```

Comment:
I know this wasn't done in this PR. But is there a reason not to pass this via the props. I think it would make this component more testable.